qemu/tests/timedrift_check_when_crash.py

```diff
@@ -90,10 +90,7 @@ def run(test, params, env):
 
     error_context.context("check time offset via ntp", logging.info)
     output = session.cmd_output(ntp_query_cmd)
-    try:
-        offset = re.findall(r"[+-](\d+\.\d+)", output, re.M)[-1]
-    except IndexError:
-        offset = 0.0
+    offset = re.findall(r"[+-]?(\d+\.\d+)", output, re.M)[-1]
```

Comment:
@YanhuiMa  If output don't include -0.00xx number.  In other words.  If re.findall fail. Maybe it will show like this error"IndexError: list index out of range".   I think you shouldn't remove 93~94 line.

Reply:
It will print the error even if I remove line 96. The reason I remove line 96 is that it will ignore errors caused by wrong pattern "r"[+-]?(\d+.\d+)"".

Reply:
I double-checked the related change. It looks good to me.

Acked:  Xiangchun Fu  xfu@redhat.com

Reply:
@YanhuiMa As we discussed, better keep the exception handling in case no offset in output. Thanks.

Reply:
@vivianQizhu Updated.
(1/1) Host_RHEL.m8.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.6.10.x86_64.myprovide.check_clock_offset.when_crash.hang: PASS (225.52 s)